homeassistant/components/zwave_js/discovery_data_template.py

```diff
@@ -180,29 +178,28 @@ def current_temperature_value(resolved_data: dict[str, Any]) -> ZwaveValue | Non
 class NumericSensorDataTemplate(BaseDiscoverySchemaDataTemplate):
     """Data template class for Z-Wave Sensor entities."""
 
-    def resolve_data(self, value: ZwaveValue) -> dict[str, Any]:
+    def resolve_data(self, value: ZwaveValue) -> str | None:
         """Resolve helper class data for a discovered value."""
-        data = {}
 
         if value.command_class == CommandClass.BATTERY:
-            data[ATTR_ENTITY_DESC_KEY] = ENTITY_DESC_KEY_BATTERY
+            return ENTITY_DESC_KEY_BATTERY
         elif value.command_class == CommandClass.METER:
             cc_specific = value.metadata.cc_specific
             meter_type_id = cc_specific[CC_SPECIFIC_METER_TYPE]
             scale_type_id = cc_specific[CC_SPECIFIC_SCALE]
             try:
                 meter_type = MeterType(meter_type_id)
             except ValueError:
-                return data
+                return None
             scale_enum = METER_TYPE_TO_SCALE_ENUM_MAP[meter_type]
             try:
                 scale_type = scale_enum(scale_type_id)
             except ValueError:
-                return data
+                return None
             for key, scale_type_set in METER_DEVICE_CLASS_MAP.items():
                 if scale_type in scale_type_set:
-                    data[ATTR_ENTITY_DESC_KEY] = key
-            data[ATTR_METER_TYPE] = meter_type
+                    return key
+            # data[ATTR_METER_TYPE] = meter_type
```

Comment:
Commented code.

Reply:
yep, will be removed after my upstream PR